doc/Makefile

```diff
@@ -16,7 +16,7 @@ help:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
-	@python settings_rstgen.py
+#	@python settings_rstgen.py
```

Comment:
remove?

Reply:
Currently due to `:recursive:` flag the `rst` files generated for `settings` are near around 70,000 so it takes more than 20 hrs locally just to read `rst` files. So we commented it out now.

First we want to ensure that the `datamodel` and `tui` works fine then we will improve the performance for `settings`.

Reply:
We have verified that we are able to generate datamodel, TUI and settings html docs locally without any error.
We have updated the code and reverted above change. Thank you.